doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -74,7 +74,9 @@ Bluetooth® LE
 Bluetooth Mesh
 --------------
 
-|no_changes_yet_note|
+* Updated:
+
+  * The Kconfig option :kconfig:option:`CONFIG_BT_MESH_DFU_METADATA_ON_BUILD` to no longer depend on the Kconfig option :kconfig:option:`CONFIG_BT_MESH_DFU_METADATA`.
```

Comment:
Maybe add something like this:
Now, by default the encoded metadata will be generated for every firmware that has the :kconfig:option:`CONFIG_BT_MESH_DFU_SRV` model enabled.
?

Reply:
That was already the case. The change it that it will no longer be generated by default if you have DFU_CLI enabled. But I can add a sentence about that.

Reply:
Ok, maybe it is fine now. When read this I feel I need more context to understand the change.

Reply:
I feel tempted to say that you maybe should add a bit more text, so it is more clear to customers what this change implies. E.g.
Now, the DFU metadata will no longer be generated by default when the Firmware Update Client model is enabled (:kconfig:option:`CONFIG_BT_MESH_DFU_CLI` set), but only when the Firmware Update Server (:kconfig:option:`CONFIG_BT_MESH_DFU_SRV`) is enabled.

Reply:
But that is a different change. There are two customer-facing changes in this PR:

1. Enabling CONFIG_BT_MESH_DFU_CLI no longer enables metadata generation by default
2. Metadata generation can now be enabled without enabling CONFIG_BT_MESH_DFU_METADATA

Should I make two bullets for these? Or combine them into one? Only mention one or the other? It seems both of your suggestions seem to conflate the two changes, but they are completely independent from each other.

Reply:
With regards to "what enables the metadata generation", that hasn't changed, other than the change mentioned in the first bullet about the DFU Client no longer enabling it by default. Shouldn't the changelog be about changes since last time?